modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -236,6 +236,58 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
 
 * Reference: link:https://neo4j.com/labs/aura-cli/1.0/cheatsheet/[Aura CLI cheetsheet^]
 
+[aura-cmi-metrics-granularity]
+== Metrics Granularity
+
+Metrics returned by integration endpoint have the basic granularity reflective of the labels exposed, namely `aggregation`, `instance_id`, `database`.
+
+An Aura instance is typically underpinned by more than one Neo4j DBMS instances to achieve availability and workload scalability. These underlying Neo4j DBMS instances typically get deployed into more than one, user selected, Cloud Provider availability zones in Primary or Secondary modes based on the workload requirements.
+
+Metrics Integration now supports a more granular view of the Aura instance metrics with additional data points for availability zone & instance mode combinations.
+
+.Example metric datapoints
+[source]
+----
+# HELP neo4j_aura_cpu_usage CPU usage (cores). CPU is used for planning and serving queries. If this metric is constantly spiking or at its limits, consider increasing the size of your instance.
+# TYPE neo4j_aura_cpu_usage gauge
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1a",instance_mode="PRIMARY"} 0.025457 1724245310000
+# HELP neo4j_aura_cpu_usage CPU usage (cores). CPU is used for planning and serving queries. If this metric is constantly spiking or at its limits, consider increasing the size of your instance.
+# TYPE neo4j_aura_cpu_usage gauge
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1b",instance_mode="PRIMARY"} 0.047088 1724245310000
+# HELP neo4j_aura_cpu_usage CPU usage (cores). CPU is used for planning and serving queries. If this metric is constantly spiking or at its limits, consider increasing the size of your instance.
+# TYPE neo4j_aura_cpu_usage gauge
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1c",instance_mode="PRIMARY"} 0.021874 1724245310000
+# HELP neo4j_aura_cpu_usage CPU usage (cores). CPU is used for planning and serving queries. If this metric is constantly spiking or at its limits, consider increasing the size of your instance.
+# TYPE neo4j_aura_cpu_usage gauge
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1a",instance_mode="SECONDARY"} 0.043957 1724245304000
+----
+
+.Additional Metric labels
+* `availability_zone` - User selected Cloud provider zone.
+* `instance_mode` - `PRIMARY` or `SECONDARY` based on user selected workload requirement. (Minimum 3 primaries per instance)
+
+.Usage
+Here is an example of gaining more insights into your Aura instance CPU usage for capacity planning:
```

Comment:
Maybe mentioning which expression language it is (assume it is Datadog?)

Reply:
It's `promtail`. Will mention that.